ironfish/src/wallet/walletdb/walletdb.ts

```diff
@@ -1023,9 +1023,11 @@ export class WalletDB {
     await this.pendingTransactionHashes.clear(tx, account.prefixRange)
   }
 
-  async cleanupDeletedAccounts(signal?: AbortSignal): Promise<void> {
-    let recordsToCleanup = 1000
+  async cleanupAllAccountsNow(signal?: AbortSignal): Promise<void> {
+    return this.cleanupDeletedAccounts(Number.MAX_SAFE_INTEGER, signal)
```

Comment:
this could be `Number.POSITIVE_INFINITY` if we want

Reply:
Yeah, I think `Number.POSITIVE_INFINITY` would be good. It aligns more with the dev expectation that it should wait for the deletion of every record